crates/sui/src/client_commands.rs

```diff
@@ -741,12 +728,9 @@ impl SuiClientCommands {
                 tx_digest,
                 profile_output,
             } => {
-                move_vm_profiler::gas_profiler_feature_disabled! {
-                    bail!(
-                        "gas-profiler feature is not enabled, rebuild or reinstall with \
-                         --features gas-profiler"
-                    );
-                };
+                if !cfg!(feature = "gas-profiler") {
+                    bail!("gas-profiler feature is not enabled, rebuild or reinstall with --features gas-profiler");
```

Comment:
Let's give proper error messages.
```suggestion
                    bail!("The gas-profiler feature is not enabled for this CLI build. Please rebuild or reinstall with --features gas-profiler.");
```